examples/3d/motion_blur.rs

```diff
@@ -7,8 +7,13 @@ use bevy::{
 };
 
 fn main() {
-    App::new()
-        .add_plugins(DefaultPlugins)
+    let mut app = App::new();
+
+    // MSAA and Motion Blur together are not compatible on WebGL
+    #[cfg(target_arch = "wasm32")]
+    app.insert_resource(Msaa::Off);
```

Comment:
if this is just for WebGL2, it should also gate on that to keep it enabled on WebGPU